src/ray/protobuf/gcs.proto

```diff
@@ -380,6 +380,8 @@ message JobsAPIInfo {
   // The node id that driver running on. It will be None only when the job status
   // is PENDING, and this field will not be deleted or modified even if the driver dies
   optional string driver_node_id = 13;
+  // The driver process exit code after driver terminates
+  optional string driver_exit_code = 14;
```

Comment:
Same here, why string instead of int

Reply:
Some test should have failed here, right? Or can a Python int silently be loaded into a string protobuf without any error? (Or is this line of code not currently run in any test?)

Reply:
It didn't fail any tests before hmm